package-lock.json

```diff
@@ -580,6 +580,21 @@
       "resolved": false,
       "integrity": "sha512-0hYQ8SB4Db5zvZB4axdMHGwEaQjkZzFjQiN9LVYvIFB2nSUHW9tYpxWriPrWDASIxiaXax83REcLxuSdnGPZtw=="
     },
+    "@cloudscape-design/browser-test-tools": {
```

Comment:
Does not look good for me. Dependabot does not run our "unlock dependencies" script. Need a follow-up to rebuild the package locally

Reply:
Good catch, I haven't thought about that! I'm following up.